app/src/main/java/com/android/ubclaunchpad/driver/session/DriverPassengersActivity.java

```diff
@@ -0,0 +1,152 @@
+package com.android.ubclaunchpad.driver.session;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.android.ubclaunchpad.driver.R;
+import com.android.ubclaunchpad.driver.user.User;
+import com.android.ubclaunchpad.driver.util.FirebaseUtils;
+import com.android.ubclaunchpad.driver.util.StringUtils;
+import com.google.firebase.database.ChildEventListener;
+import com.google.firebase.database.DataSnapshot;
+import com.google.firebase.database.DatabaseError;
+import com.google.firebase.database.DatabaseReference;
+import com.google.firebase.database.ValueEventListener;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DriverPassengersActivity extends AppCompatActivity {
+
+    private static final String TAG = DriverPassengersActivity.class.getSimpleName();
+
+    private DatabaseReference session;
+    private ChildEventListener listener;
+    private DriverPassengersAdapter adapter;
+    private List<User> passengers;
+    Button startButton;
+    TextView driverName;
+    TextView driverStatus;
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_driver_passengers);
+        startButton = (Button) findViewById(R.id.start_button);
+
+        String sessionName = (String) getIntent().getBundleExtra("firebaseData").get(StringUtils.FirebaseSessionName);
+        String driverUid = (String) getIntent().getBundleExtra("firebaseData").get("driverUid");
+
+        driverName = (TextView) findViewById(R.id.driver_name);
```

Comment:
you can use ButterKnife for this, for example @BindView instead of findViewById()